src/aria/utils/Array.js

```diff
@@ -173,6 +173,31 @@ var Aria = require("../Aria");
                     }
                 }
                 return output;
+            },
+            /**
+             * This tests whether all elements in the array pass the test given by the predicate callback function. 
+             * As long as the predicate (callback) returns truthy values, the loop will continue. However, as soon as a falsy
+             * value is returned, the function will return without analyzing the rest of the elements.
+             * @param {Array} array Array to analyze.
```

Comment:
Just a remark: `analyzing` and `analyze` are maybe a bit specific words: the array and its items can be ___processed___ without being _analyzed_.